recipe/bld.bat

```diff
@@ -0,0 +1,10 @@
+nmake -f psi\msvc.mak GSROOTDIR=%LIBRARY_PREFIX% WIN64= MSVC_VERSION=%VS_MAJOR% DEVSTUDIO=
+if errorlevel 1 exit 1
+
+move bin\*.dll %LIBRARY_BIN%
+cp bin\gswin64c.exe %LIBRARY_BIN%\gs.exe
```

Comment:
I'm not 100% sure my moving and copying here is correct. Specially the `gswin64c.exe` to `gs.exe`. It seems to work for me locally though.

Reply:
I think you also need a `move bin\*.lib %LIBRARY_LIB%`. Just looking at the Linux package and it seems that the contents of the source `lib` dir has been copied to `share/ghostscript/9.22/lib` - not sure if we need to do this also. There appears to be more in the `bin` dir on Linux also which is a bit weird.
Do you know if there should be an `include` directory? Doesn't seem to be on Linux - maybe applications never link to the lib? I don't have any experience with this library, but we might only find what is missing once we try and use it elsewhere...

Reply:
> I think you also need a `move bin\*.lib %LIBRARY_LIB%`.

That is done in the line below, right?

> Just looking at the Linux package and it seems that the contents of the source `lib` dir has been copied to `share/ghostscript/9.22/lib` - not sure if we need to do this also.

If we don't have to I prefer to not do it. Adding versioned paths is always a headache.

> There appears to be more in the `bin` dir on Linux also which is a bit weird. Do you know if there should be an `include` directory? Doesn't seem to be on Linux - maybe applications never link to the lib? I don't have any experience with this library, but we might only find what is missing once we try and use it elsewhere...

To be honest I never checked this package the way I should and the only package depending on it is `GMT`. @leouieda do you need the header files? If I'm not mistaken all that GMT does is to call the binary, right?

Reply:
> That is done in the line below, right?

Sorry - I missed that. Looks all good to me :smile:

Reply:
> @leouieda do you need the header files?

No, just the executable. GMT makes system calls to it.